package.json

```diff
@@ -86,6 +86,7 @@
     "mysql": "^2.17.1",
     "npm": "^6.13.4",
     "oracledb": "^4.2.0",
+    "paraphrase": "^1.8.0",
```

Comment:
Can we pin this since we _know_ this specific version caused this bug?

Reply:
Good idea, done!